packages/url_launcher/url_launcher_macos/lib/url_launcher_macos.dart

```diff
@@ -0,0 +1,3 @@
+// The url_launcher_platform_interface defaults to MethodChannelUrlLauncher 
+// as its instance, which is all the macOS implementation needs. This file
+// is here to silence warnings when publishing to pub.
```

Comment:
IMO we should make this comment into a TODO (to not require this file), file an issue against pub and reference it here.